codes.json

```diff
@@ -57,17 +57,18 @@
   }
 ,
   {
-    "title" : "MADX",
-    "subtitle" : "Lie algebraic beam dynamics code with space charge",
-    "description" : "MaryLie/IMPACT (ML/I) combines features of MaryLie and IMPACT along with other features including a MAD-like front end",
-    "homepage" : "TBD",
-    "institutions" : ["Lawrence Berkeley National Laboratory", "University of Maryland"],
-    "institution_link" : "https://www.lbl.gov",
-    "repository" : "https://github.com/...",
-    "issue_tracker" : "https://github.com/...",
-    "license" : "TBD",
-    "publication" : "A paper citation",
-    "publication_link" : "https://doi.org/DOIHERE"
+        "title" : "MAD8 and MAD-X (includes PTC)",
+    "subtitle" : "Methodical Accelerator Design",
+    "description" : "MAD-X is a general-purpose tool for charged-particle optics design and studies in alternating-gradient accelerators and beam lines. It can handle medium size to very large accelerators and solves various problems on such machines.  MAD-X is the successor of MAD-8 and was specifically adapted to the needs of the design of the LHC. The PTC library of E. Forest is also embedded in MAD-X as an addition to better support small and low energy accelerators. ",
+    "homepage" : "http://mad.web.cern.ch/mad/",
+    "institutions" : ["Conseil Européen pour la Recherche Nucléaire (CERN)"],
+    "institution_link" : "https://home.cern/",
+    "repository" : "https://github.com/MethodicalAcceleratorDesign/MAD-X",
+    "issue_tracker" : "https://github.com/MethodicalAcceleratorDesign/MAD-X/issues",
+    "license" : "https://github.com/MethodicalAcceleratorDesign/MAD-X/blob/master/License.txt",
```

Comment:
Although there is no standardized abbreviation ([SPDX identifier](https://en.wikipedia.org/wiki/Software_Package_Data_Exchange)) for this license, we can name id "SSDS" as it is also called in the repo.

Reply:
Fixed in #11 :-)